src/structures/InteractionCollector.js

```diff
@@ -33,21 +33,28 @@ class InteractionCollector extends Collector {
 
     /**
      * The message from which to collect interactions, if provided
-     * @type {?Message}
+     * @type {?Snowflake}
      */
-    this.message = options.message ?? null;
+    this.messageId = options.message?.id;
```

Comment:
```suggestion
    this.messageId = options.message?.id ?? null;
```
Set it to null if not provided (as per typings), don't leave it undefined.